examples/CreateSimpleEncoding.php

```diff
@@ -67,4 +67,8 @@
 $jobConfig->outputFormat[] = new HlsOutputFormat();
 
 // RUN JOB AND WAIT UNTIL IT HAS FINISHED
-$client->runJobAndWaitForCompletion($jobConfig);
\ No newline at end of file
+$jobContainer = $client->runJobAndWaitForCompletion($jobConfig);
+foreach($jobContainer->encodingContainers as $encodingContainer)
```

Comment:
Probably add an empty line and a comment that this prints the detailed encoding status information.